docs/application/tizen-studio/release-notes/4-6-release-notes.md

```diff
@@ -0,0 +1,125 @@
+| UI   App(Main)         | M                  | M                       | M                      | 0                         | M                      | M                      | 0                     |
+| Service   App(Main)    | 0                  | 0                       | 0                      | 0                         | M                      | M                      | 0                     |
+| Widget   App(Main)     | 0                  | 0                       | 0                      | 0                         | M                      | M                      | 0                     |
+| Component   App(Main)  | 0                  | 0                       | 0                      | 0                         | M                      | M                      | 0                     |
+| Shared   Library(Main) | 0                  | 0                       | 0                      | 0                         | M                      | M                      | 0                     |
+| Static   Library(Main) | 0                  | 0                       | 0                      | 0                         | M                      | M                      | 0                     |
+| Watch   App(Main)      | 0                  | M                       | 0                      | 0                         | M                      | M                      | 0                     |
+
+**Web Apps**
+
+| Apps               | Native Service App (Dependent) | Native Widget App (Dependent) | Web App (Dependent) | Web Widget (Dependent) | Dotnet UI app (Dependent) | Dotnet Service App (Dependent) |   |
+|--------------------|--------------------------------|-------------------------------|---------------------|------------------------|---------------------------|--------------------------------|---|
+| Web   App(Main)    | M                              | M                             | 1                   | M                      | M                         | M                              |   |
+| Web   Widget(Main) | 0                              | 0                             | 0                   | 0                      | 0                         | 0                              |   |
+
+
+  **Hybrid App**
+  Developers can now have multiple dependent applications **of different types** in a single workspace and perform all App life cycle events like app creation, building, packaging, installing and testing.
+    
+| Main App          | Possible Sub App Combinations |
+|-------------------|-------------------------------|
+| Native   Main App | Native + Dotnet               |
+| Dotnet   Main App | Dotnet + Native               |
+| Web   Main App    | Web + Native +   Dotnet       |
+
+- CLI
+  - Added support for importing Hybrid and Multi projects
```

Comment:
To maintain consistency with other bullet points, the sentence should end with a period. 
```suggestion
  - Added support for importing Hybrid and Multi projects.
```